share/picongpu/examples/FoilLCT/include/picongpu/simulation_defines/param/density.param

```diff
@@ -102,5 +102,51 @@ namespace densityProfiles
     /* definition of free formula profile */
     using FlatFoilWithRamp = FreeFormulaImpl< FlatFoilWithRampFunctor >;
 
+    struct ProbeEveryFourthCellFunctor
```

Comment:
Can we implement it as `ProbeEveryNthCellFunctor`.

```C++
template< uint32_t T_N >
struct ProbeEveryNthCellFunctor;
```

Reply:
sure, I can also make it
```C++
template<
    uint32_t T_Nx, 
    uint32_t T_Ny,
    uint32_t T_Nz
>
struct ProbeEveryNthCellFunctor;
```

But in both cases, it might be too complicated for user code and I should maybe move it to a general particle density "profile" and just use it. What do you think?

Reply:
Yes a general profile would be a good idea. Please use `CT::vector<...>` instead of three numbers to allow the usage for 2 and 3 dimensional densities.